instrumentation/opentelemetry/span.go

```diff
@@ -31,6 +31,14 @@ func (l *AttributeList) GetValue(key string) interface{} {
 	return nil
 }
 
+func (l *AttributeList) GetAll() []sdk.Attribute {
+	attributes := make([]sdk.Attribute, len(l.attrs))
+	for _, attr := range l.attrs {
+		attributes = append(attributes, sdk.Attribute{Key: string(attr.Key), Value: attr.Value.AsInterface()})
```

Comment:
Oh this is tricky in golang. I think it will produce an array with twice the intended length and with first of the array full of empty values. append does not start at index 0.

Reply:
Thanks, I thought the size we specify was just for memory allocation, didnt realise we get a filled slice. Updated it